src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/NotifyLiPClaimantHwFOutcomeHandler.java

```diff
@@ -0,0 +1,117 @@
+package uk.gov.hmcts.reform.civil.handler.callback.camunda.notification;
+
+import com.google.common.collect.ImmutableMap;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse;
+import uk.gov.hmcts.reform.civil.callback.Callback;
+import uk.gov.hmcts.reform.civil.callback.CallbackHandler;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.callback.CaseEvent;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.notify.NotificationService;
+import uk.gov.hmcts.reform.civil.notify.NotificationsProperties;
+import uk.gov.hmcts.reform.civil.service.FeatureToggleService;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_SUBMIT;
+import static uk.gov.hmcts.reform.civil.utils.PartyUtils.getPartyNameBasedOnType;
+
+@Service
+@RequiredArgsConstructor
+public class NotifyLiPClaimantHwFOutcomeHandler extends CallbackHandler implements NotificationData {
+
+    private static final List<CaseEvent> EVENTS = List.of(CaseEvent.NOTIFY_LIP_CLAIMANT_HWF_OUTCOME);
+    public static final String TASK_ID = "NotifyClaimantHwFOutcome";
+    private static final String REFERENCE_TEMPLATE = "hwf-outcome-notification-%s";
+    private final NotificationService notificationService;
+    private final NotificationsProperties notificationsProperties;
+    private final FeatureToggleService toggleService;
+
+    private final Map<String, Callback> callBackMap = Map.of(
+        callbackKey(ABOUT_TO_SUBMIT), this::notifyApplicantForHwFOutcome
+    );
+    private Map<CaseEvent, String> emailTemplates;
+
+    @Override
+    protected Map<String, Callback> callbacks() {
+        return callBackMap;
+    }
+
+    @Override
+    public String camundaActivityId(CallbackParams callbackParams) {
+        return TASK_ID;
+    }
+
+    private CallbackResponse notifyApplicantForHwFOutcome(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+
+        if (caseData.isLipvLipOneVOne() && toggleService.isLipVLipEnabled()) {
```

Comment:
Do we need this validation? Asking bcoz these events are used only for LiP claimants so the notification handler.

Reply:
Well spotted, done!